### ISSUE TITLE
Bad fd number error (Ubuntu GNOME 15.04)

### ISSUE THREAD

**Issue by mariellep** [x89/Tea#19]
./tea.sh: 59: ./tea.sh: Syntax error: Bad fd number

Don't know if it's distro-specific (although I highly doubt it!) and it's obviously not a compiler problem.

**Comment by x89** [comment]
It might be an issue with `dash` - is that what your `/bin/sh` points to?

**Comment by mariellep** [comment]
I don't think so. How would I find out?



Sent using CloudMagic [https://cloudmagic.com/k/d/mailapp?ct=pa&cv=6.3.33&pv=4.4.2]

On Fri, May 29, 2015 at 4:53 PM, x89 < notifications@github.com [notifications@github.com] > wrote:
It might be an issue with dash - is that what your /bin/sh points to?

**Comment by anlutro** [comment]
```
andreas@viserys ~
$ ls -l $(which sh)
lrwxrwxrwx 1 root root 4 Nov  8  2014 /bin/sh -> dash*
```

**Comment by mariellep** [comment]
It points to dash, but without the *

Could it possibly have anything to do with the $PATH? I had to change some stuff
in order to get some libraries to work.



Sent using CloudMagic [https://cloudmagic.com/k/d/mailapp?ct=pa&cv=6.3.33&pv=4.4.2]

On Fri, May 29, 2015 at 10:02 PM, Andreas Lutro < notifications@github.com [notifications@github.com] > wrote:
andreas@viserys ~
$ ls -l $(which sh)
lrwxrwxrwx 1 root root 4 Nov  8  2014 /bin/sh -> dash*

**Comment by mariellep** [comment]
Aha! It was dash. I fixed it by running

sudo dpkg-reconfigure dash

and removing the symbolic link.

On Fri, May 29, 2015 at 10:10 PM, Marielle Pinheiro <
marielle.pinheiro@gmail.com> wrote:

> It points to dash, but without the *
>
> Could it possibly have anything to do with the $PATH? I had to change some
> stuff in order to get some libraries to work.
>
> Sent using CloudMagic
> <https://cloudmagic.com/k/d/mailapp?ct=pa&cv=6.3.33&pv=4.4.2>
> On Fri, May 29, 2015 at 10:02 PM, Andreas Lutro <notifications@github.com>
> wrote:
>
> andreas@viserys ~
> $ ls -l $(which sh)
> lrwxrwxrwx 1 root root 4 Nov  8  2014 /bin/sh -> dash*
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/x89/Tea/issues/19#issuecomment-106974224>.
>
>


-- 
There is a theory which states that if ever anyone discovers exactly what
the Universe is for and why it is here, it will instantly disappear and be
replaced by something even more bizarre and inexplicable.

There is another theory which states that this has already happened.

-The Restaurant at the End of the Universe

**Comment by anlutro** [comment]
You shouldn't do that, it might break your system. I can fix it for dash.

**Comment by anlutro** [comment]
If you `git pull` or re-download now, it should work fine.

**Comment by mariellep** [comment]
Yay! Thanks.



Sent using CloudMagic [https://cloudmagic.com/k/d/mailapp?ct=pa&cv=6.3.33&pv=4.4.2]

On Fri, May 29, 2015 at 10:23 PM, Andreas Lutro < notifications@github.com [notifications@github.com] > wrote:
Closed #19 [https://github.com/x89/Tea/issues/19] .
